README.md

```diff
@@ -32,6 +33,19 @@ query = server.query()
 print(f"The server has the following players online: {', '.join(query.players.names)}")
 ```
 
+Bedrock Edition
+```python
+from mcstatus import MinecraftBedrockServer
+
+# Use MineCraftBedrockServer for Bedrock servers. If you do not specify a port, 19132 will be used by default.
+server = MinecraftBedrockServer(host="example.org", port=19132)
```

Comment:
Show the `lookup` method as well as the default constructor, just like in the java version example
```suggestion
# If you know the host and port, you may skip this and use MinecraftBedrockServer("example.org", 19132)
server = MinecraftBedrockServer.lookup("example.org:19132")
```